docs/how-to/ccv0.sh

```diff
@@ -467,7 +467,7 @@ run_kata_and_capture_logs() {
 
 get_ids() {
     guest_cid=$(sudo ss -H --vsock | awk '{print $6}' | cut -d: -f1)
-    sandbox_id=$(ps -ef | grep qemu | egrep -o "sandbox-[^,][^,]*" | sed 's/sandbox-//g' | awk '{print $1}')
+    sandbox_id=$(ps -ef | grep containerd-shim-kata-v2 | egrep -o "id [^,][^,].* " | awk '{print $2}')
```

Comment:
This looks correct if you want to make it hypervisor-agnostic. At least I don't know if there is a better way.

Keep in mind that assume you have only one shim process running...which might not be true. To be fair, the old version of the code had the same assumption.

Reply:
Exactly.  The old code was assuming only one QEMU process running.
And if you happen to know a better way to do so, I'm here to learn. :-)